src/crc32c/LICENSE

```diff
@@ -0,0 +1,28 @@
+Copyright 2017, The CRC32C Authors.
```

Comment:
A `contrib/debian/copyright` entry is needed for crc32c. It should look something like this (modelled after leveldb, since the license and the convention for listing authors is the same):
```
Files: src/crc32c/*
Copyright: 2017 The CRC32C Authors
License: BSD-3-clause-Google
Comment: The CRC32C Authors are listed in src/crc32c/AUTHORS.
```

Reply:
This can be done in the v5.2.0 release PR.